installer/oss-kits/docker-oss-test.sh

```diff
@@ -21,4 +23,4 @@ if [ "$(uname -m)" == "x86_64" ]; then
 else
     echo "  Only 64-bit systems are supported. Docker agent will not be installed." 1>&2
     exit 1
-fi
\ No newline at end of file
+fi
```

Comment:
I guess it doesn't really matter; this doesn't look like one of the scripts that's super picky.